frontend/components/GoverningDocuments/DocumentButton.tsx

```diff
@@ -14,6 +14,7 @@ import { useDeleteGoverningDocumentMutation } from '~/generated/graphql';
 import { useApiAccess } from '~/providers/ApiAccessProvider';
 import Link from '../Link';
 import routes from '~/routes';
+import packageJson from '../../package.json';
```

Comment:
That's a neat solution, but I would suggest using package-lock.json instead to ensure it's the right version.